supervision/tracker/byte_tracker/core.py

```diff
@@ -261,6 +260,13 @@ def callback(frame: np.ndarray, index: int) -> np.ndarray:
 
         return detections
 
+    def reset(self):
```

Comment:
Can we document this method? For the method to be displayed in docs it needs to be public and have docstring.

Reply:
Hi @SkalskiP Thank you for the suggestions. I have already implemented the changes. Please let me know if there is anything else that needs to be updated. 🥷 🚀